Makefile

```diff
@@ -71,53 +73,55 @@ help:
 	@echo " DB_DIALECT:\t\t$(.DB_DIALECT)"
 	@echo ""
 
-dev-image:
-	docker build -t $(.DEV_IMAGE) .
+image-base:
+	docker build .
+
+image-local: image-base
+	docker build -t $(DOCKER_IMAGE_LOCAL) . -f env.local.Dockerfile
 
-serve-image:
-	docker build -t $(.SERVE_IMAGE) . -f serve.Dockerfile
+image-prod: image-base
+	docker build -t $(DOCKER_IMAGE_PROD) . -f env.prod.Dockerfile
 
-build: dev-image
+build: image-local
 	mkdir -p $(PWD)/build
 	docker run \
 		--rm \
 		-it \
-		-v $(PWD)/build:/www/build \
 		$(.SHARED_VOLUMES) \
 		$(.ENV_VARIABLES) \
 		--entrypoint=npm \
-		$(.DEV_IMAGE) run build
+		$(DOCKER_IMAGE_LOCAL) run build
 
-test: dev-image
+test: image-local
 	docker run \
 		--rm \
-		--name sa-test \
+		--name $(DOCKER_IMAGE_ALIAS)-test \
 		-it \
 		$(.SHARED_VOLUMES) \
-		$(.ENV_VARIABLES) \
+		$(.ENV_VARIABLES) \	

```

Comment:
I know this will sound weird, but if you try command it will give error because slash should be EOL and nothing should follow it :) but it is very good changes

Reply:
fixed